index.html

```diff
@@ -74,38 +82,46 @@ <h4 class="head-color roboto text-normal align-left">Skills</h4>
             </li>
           </ul>
         </div>
-        <hr class="horizontal" />
-        <div class="socials">
-          <a href="https://github.com/itsmraga-hub"
-            ><img src="assets/icons/github.png" alt="github logo"
-          /></a>
-          <a href="https://github.com/itsmraga-hub"
-            ><img src="assets/icons/linkedin.png" alt="linkedin logo" />
-          </a>
-          <a href="">
-            <img src="assets/icons/fingers.png" alt="two fingers held up" />
-          </a>
-          <a href="https://twitter.com/RagaMacharia"
-            ><img src="assets/icons/twitter.png" alt="teitter logo"
-          /></a>
-          <a href="">
-            <img src="assets/icons/M.png" alt="letter M" />
-          </a>
-        </div>
-        <div class="nav-arrow">
-          <a href="#work"
-            ><img class="arrow bow" src="assets/projects/arrow.png" alt="" />
-          </a>
-          <a href="#work" class="works"
-            ><p class="work roboto align-center">Work</p></a
-          >
+        <div class="navigation-space">
+          <div class="nav-arrow-1 nav-arrow">
+            <a href="#work"
+              ><img class="arrow bow" src="assets/projects/arrow.png" alt="" />
+            </a>
+            <a href="#work" class="works"
+              ><p class="work roboto align-center">Work</p></a
+            >
+          </div>
+          <div class="socials">
+            <a href="https://github.com/itsmraga-hub"
+              ><img src="assets/icons/github.png" alt="github logo"
+            /></a>
+            <a href="https://github.com/itsmraga-hub"
+              ><img src="assets/icons/linkedin.png" alt="linkedin logo" />
+            </a>
+            <a href="">
+              <img src="assets/icons/fingers.png" alt="two fingers held up" />
+            </a>
+            <a href="https://twitter.com/RagaMacharia"
+              ><img src="assets/icons/twitter.png" alt="teitter logo"
+            /></a>
+            <a href="">
+              <img src="assets/icons/M.png" alt="letter M" />
+            </a>
+          </div>
+          <div class="nav-arrow">
+            <a href="#work"
+              ><img class="arrow bow" src="assets/projects/arrow.png" alt="" />
+            </a>
+            <a href="#work" class="works"
+              ><p class="work roboto align-center">Work</p></a
+            >
+          </div>
```

Comment:
- [OPTIONAL] Great job here 👌 but I think it would be better to use `<ul>` and `<li>` instead of  `<div>` to follow the best practice 👌💯

> For example 
```
<ul  class="socials">
   <li><a href="https://github.com/itsmraga-hub"><img src="assets/icons/github.png" alt="github logo"/></a</li>
   ........
</ul>
```